src/sass/static/virtual_acc_opening.scss

```diff
@@ -0,0 +1,36 @@
+#virtual-form{
+	text-align: center;
```

Comment:
why so much custom CSS? Please try to use only our standard styling from https://www.binary.com/styles